### ISSUE TITLE
Path resolver fails for Sass/Scss?

### ISSUE THREAD

**Issue by nnx0r** [aziz/SublimeHyperClick#35]
This might be a question how to, but also might be a bug. I don't really know, so I am asking here.
But how do we resolve path for Sass/Scss without extension being present?

```scss
@import 'layout.scss'; // it works
@import 'layout'; // it fails (but file obviously exists)
```

**Comment by cliener** [comment]
At a guess, the fault may lie in the [path resolver](https://github.com/aziz/SublimeHyperClick/blob/master/hyper_click/path_resolver.py) which has the path for SCSS hard-coded to `sass` i.e. `elif lang == 'sass':`
If I knew more about python and Package Control, I'd submit a PR.

**Comment by cristianl** [comment]
Fixed in https://github.com/aziz/SublimeHyperClick/pull/53.

**Comment by cristianl** [comment]
Shipped in [1.10.0](https://github.com/aziz/SublimeHyperClick/releases/tag/1.10.0)
